locale/ar/LC_MESSAGES/django.po

```diff
@@ -105,27 +105,27 @@ msgid "Welcome to %(server)s, %(member name)s!\n"
 "INFO: Get this message again\n"
 "\n"
 "You can also message me on Telegram: %(telegram link)s"
-msgstr "مرحبا بك في member name)s! ,%(server)s)%\n"
+msgstr " مرحبا %(s(member name في, %(s(server\n"
```

Comment:
This looks wrong.

If you merge my PRs you'll get additional locale tests that catch broken tokens like this.